src/main/java/uk/gov/hmcts/reform/amapi/controllers/RootController.java

```diff
@@ -14,27 +17,11 @@
 @RestController
 @SuppressWarnings("PMD")
```

Comment:
We may wanna remove pmd rules suppresion now that we are starting to implement actual stories